docs/design/feedback.rst

```diff
@@ -0,0 +1,119 @@
+  
+- Include clear call to actions that guide the User on how to proceed. For example:
+  
+  'No Email activity? Start sending some Campaigns to populate this data!'
+  
+- If the lack of data is due to incomplete configuration, provide direct guidance:
+  
+  'It looks like device tracking isn't active. Go to settings to turn it on.'
+  
+- Help Users understand the value of the missing data:
+  
+  'Device information helps optimize your Campaigns for different platforms. Once this data becomes available, see detailed analytics here.'
+  
+- Use icons, colors, or visual elements to indicate areas that need attention.
+
+This approach not only improves immediate usability but also accelerates the Users' learning curve, leading to more sophisticated use of the platform over time. Users don't feel 'stuck' when encountering areas without data, but are instead motivated to explore and fill those gaps.
+
+.. vale off
+
+'No Results' template
+---------------------
+
+.. vale on
+
+Mautic includes a reusable template for displaying informative messages when no results are available. This template offers a consistent and flexible way to provide User feedback, with options for additional actions.
+
+Template structure
+------------------
+
+.. code-block:: twig
+
+    {% if tip is defined %}
+    <div class="alert alert-info">
+        {{ tip|trans }}
+        {% if link is defined and (href is defined or onclick is defined) %}
+        <a class="ml-a" href="{{href}}" onclick="{{onclick}}">{{link|trans}}</a>
+        {% endif %}
+    </div>
+    {% endif %}
+
+Parameters
+----------
+
+The template accepts the following parameters:
+
+.. vale off
+
+- **tip** (required): display a translation string that contains the main message.
+- **link** (optional): a translation string for the link/button text.
+- **href** (optional): use the ``URL`` for navigation when clicking the link.
+- **onclick** (optional): JavaScript function to be executed when the link is clicked.
+
+.. vale on
+
+Functionality
+-------------
+
+If you've defined ``tip``, a ``div`` with the class ``alert alert-info`` that contains the translated message displays, if not then nothing renders. If you've defined `link` and at least one of ``href`` or ``onclick``, a link below the main message displays. Configure the link to navigate to a new page with ``href`` or execute a JavaScript function with ``onclick``.
+
+Usage example
+-------------
+
+To use this template in your code, you can include it as follows:
+
+.. code-block:: twig
+
+    {{ include('@MauticCore/Helper/no-information.html.twig', {
+        'tip': 'Mautic.segment.no.results',
+        'link': 'Mautic.segment.add.new',
+        'href': '{{ path('Mautic_segment_action', {'objectAction': 'new'}) }}'
+    }) }}
+
+In this example, the template displays a message indicating that no Segments are available, with a link to create a new Segment.
+
+Why?
```

Comment:
Headings need to be unique, perhaps we should expand this to 'Why provide a uniform message in the absence of results?'?